backend/spec/requests/api/v1/participant/studies_spec.rb

```diff
@@ -144,6 +145,14 @@
         api_get 'participant/studies'
         expect(response_hash[:data]).not_to include(a_hash_including(id: study1.id))
       end
+
+      it 'excludes completed launched studies' do
+        # End both sessions
+        study2.end
+        study2.end
```

Comment:
this confused me for a bit, might be better to do:
`study2.stages.each{|stage| stage.update!(status: 'completed') }`

Reply:
I would prefer to keep using `study.end` as it will uncover any issues if we change the `end` method down the line - not sure how strongly you feel about it either way though

the end method:
```
  def end
    stages.where.not(status: 'completed').first&.update!(status: 'completed')
  end
```